lib/Configuration.php

```diff
@@ -54,6 +54,7 @@ class Configuration
             'icon'                     => 'identicon',
             'cspheader'                => 'default-src \'none\'; manifest-src \'self\'; connect-src *; script-src \'self\'; style-src \'self\'; font-src \'self\'; img-src \'self\' data:; referrer no-referrer; sandbox allow-same-origin allow-scripts allow-forms allow-popups',
             'zerobincompatibility'     => false,
+            'webserver'     => 'Apache',
```

Comment:
We can fix the indentation afterwards (PHP-CS does it for us), but if you want, you can of course bring it in a nicely, order "table style"…